code/modules/antagonists/_common/antag_datum.dm

```diff
@@ -231,6 +231,7 @@ GLOBAL_LIST_EMPTY(antagonists)
 /datum/antagonist/proc/greet()
 	if(!silent)
 		to_chat(owner.current, span_big("You are \the [src]."))
+		to_chat(owner.current, span_infoplain("<span class='doyourjobidiot'>Remember that being an antagonist does not exclude you from the server rules regarding RP standards.")) //SKYRAT EDIT - RP REMINDER
```

Comment:
I'd like you to make that `doyourjobidiot` class a better name and to make it a define, no ugly spans in there please.

Reply:
fixed